types/block.go

```diff
@@ -748,9 +748,9 @@ func (ecs ExtendedCommitSig) ValidateBasic() error {
 		if len(ecs.Extension) > MaxVoteExtensionSize {
 			return fmt.Errorf("vote extension is too big (max: %d)", MaxVoteExtensionSize)
 		}
-		if len(ecs.ExtensionSignature) > MaxSignatureSize {
-			return fmt.Errorf("vote extension signature is too big (max: %d)", MaxSignatureSize)
-		}
+		// if len(ecs.ExtensionSignature) > MaxSignatureSize {
```

Comment:
You may want to restore this "if" as well

Reply:
Oops missed this!

Reply:
Thanks! Resolved